### DISCUSSION TITLE
Tab Panes

### DISCUSSION THREAD

**Discussion by animalogic** [coreui/coreui-react#370] [Q&A]
https://coreui.io/react/docs/components/navs-tabs/#tab-panes

We are working with the Pro React Admin Template - v.4.7.0 - and the Tab Panes <CNavLink> are opening a new page instead of displaying the content pane.

https://coreui.io/demos/react/4.6/default/#/base/navs

We noticed the Admin demo for React and Vue do not show a working version of the Tab Panes - while there are working examples on the demos for Angular and Bootstrap.

Can anyone provide a working example of the <CNavLink> working with a content pane?

**Comment by mrholek** [comment]
@animalogic I'm looking at where is the issue because in the development environment everything works well.

Sometimes some examples are not available on templates (we do our best to have everything in templates :)) that's why I recommend using our documentation.

**Reply by animalogic** [reply]
@mrholek thank you for updating the documentation

**Reply by mrholek** [reply]
You're welcome!
